shared/api/certificate.go

```diff
@@ -72,3 +77,36 @@ type Certificate struct {
 func (cert *Certificate) Writable() CertificatePut {
 	return cert.CertificatePut
 }
+
+// ClientJoinToken represents the fields contained within an encoded client join token.
+//
+// swagger:model
+//
+// API extension: client_join_token
+type ClientJoinToken struct {
```

Comment:
Probably should be prefixed with `Certificate` for consistency.

So maybe `CertificateAddToken`?